src/main/kotlin/org/rust/lang/core/resolve/Processors.kt

```diff
@@ -511,3 +514,41 @@ fun filterDeriveProcMacros(processor: RsResolveProcessor): RsResolveProcessor =
         if (!function.isCustomDeriveProcMacroDef) return@wrapWithFilter false
         true
     }
+
+fun RsOuterAttributeOwner.shouldHideElementInCompletion(contextMod: RsMod): Boolean {
```

Comment:
Btw interesting fact that there can be stable item inside unstable module - `core::unicode::UNICODE_VERSION`